### PR TITLE
Update quilljs format to Reaction report

### PR DESCRIPTION
**Pull request by CamAnNguyen**
- Update sablon docx template
- Add a quill delta to HTML converter
- Apply quill format to report

### CODE REVIEW

File: app/assets/javascripts/components/QuillEditor.js [hunk 1]
```diff
@@ -17,17 +17,17 @@ export default class QuillEditor extends React.Component {
...
           [{ 'list': 'ordered'}, { 'list': 'bullet' }],
           [{ 'script': 'sub'}, { 'script': 'super' }],
-          [{ 'indent': '-1'}, { 'indent': '+1' }],
-          // [{ 'direction': 'rtl' }],
-          // [{ 'size': ['small', false, 'large', 'huge'] }],
           [{ 'header': [1, 2, 3, 4, 5, 6, false] }],
           [{ 'color': [] }, { 'background': [] }],
-          [{ 'font': [] }],
-          [{ 'align': [] }],
+          // [{ 'font': [] }],
```
Comment (Sergeykot) [inline app/assets/javascripts/components/QuillEditor.js]: odd commened code

**Comment by Sergeykot** [comment]
needs a small cleanup ;)

File: app/assets/javascripts/components/QuillEditor.js (hunk 1 above)
Comment (CamAnNguyen) [inline app/assets/javascripts/components/QuillEditor.js]: We can uncomment a single line to add new format to the editor so that I don't use block comment

Comment (Sergeykot) [inline app/assets/javascripts/components/QuillEditor.js]: ok than we could leave it.

**Pull request merged**